stratum/hal/lib/barefoot/bf_chassis_manager.h

```diff
@@ -73,6 +73,9 @@ class BFChassisManager {
   ::util::StatusOr<int> GetUnitFromNodeId(uint64 node_id) const
       SHARED_LOCKS_REQUIRED(chassis_lock);
 
+  ::util::StatusOr<uint32> GetSdkPortId(uint64 node_id, uint32 port_id) const
+      SHARED_LOCKS_REQUIRED(chassis_lock);
+
```

Comment:
Must be virtual for the mock (which we don't have right now).

Reply:
Can also be private, there are no calls from outside this class.

Reply:
I was thinking that we want to keep this public to enable future calls (e.g. to enable port translation in P4RT). But we can also make this public in the future if needed.